pandas/formats/format.py

```diff
@@ -2260,6 +2260,35 @@ def _format_strings(self):
         return fmt_values
 
 
+def format_percentiles(percentiles):
+    '''
+    Outputs rounded and formatted percentiles. Rounding preserves uniqeness.
+
+    Parameters
+    ----------
```

Comment:
have a Returns section that say what you are guaranteeing on the return (e.g. list of increasing order, formatted)